prow/cmd/deck/main.go

```diff
@@ -50,23 +50,24 @@ var objReg = regexp.MustCompile(`^[\w-]+$`)
 func main() {
 	flag.Parse()
 	logrus.SetFormatter(&logrus.JSONFormatter{})
+	logger := logrus.WithField("component", "deck")
```

Comment:
Also in a follow-up a utility func was requested for this boilerplate at the top to initialize the logger.